challenge_2/python/tavery321/src/c2fix.py

```diff
@@ -0,0 +1,12 @@
+a = ['a', 'b', 'c', 2, 3, 5, 'a', 2, 'c', 5, 'b']
```

Comment:
For lists you only use as a set, where you only add, remove, test for membership, and don't need the elements to stay in a certain order, it would be better to use a set or dict instead.